modules/samples/producers/video-frame-client/src/main/resources/log4j.properties

```diff
@@ -0,0 +1,33 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
Changed in https://github.com/hemikak/product-cep/commit/6210d26b510acc5fc699d74e24cdcd17d3c8dd1d